parent/pom.xml

```diff
@@ -63,6 +65,12 @@
                 <groupId>org.apache.santuario</groupId>
                 <artifactId>xmlsec</artifactId>
                 <version>${xmlsec.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.sun.activation</groupId>
+                        <artifactId>jakarta.activation</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Should this be contributed upstream to Apache Santuario?

Reply:
Yes. I'll send another PR to fix this in Santuario too.